wsgi/openshift/services/models.py

```diff
@@ -1,8 +1,35 @@
 from django.db import models
+import json
+import requests
```

Comment:
This needs to be added to the `setup.py` without qualifying the version. In a brand new virtual environment, wsgi grumps that it is missing when trying to start the site.